util/device/registerDevice_test.go

```
@@ -0,0 +1,455 @@
+		// ID will be set by GORM upon creation
+		Uuid:         uuid.New(),
+		FirstName:    "Global",
+		LastName:     "Tester",
+		OIB:          "12345678901", // Ensure this is unique for each created user in tests
+		Email:        "global.tester@example.com",
+		PasswordHash: "hashedpassword",
+		Role:         model.RoleOsoba,
+		BirthDate:    time.Now().AddDate(-30, 0, 0),
+		Residence:    "123 Test St",
+	}
+	testDeviceTokenGlobal = "fixed-test-device-token"
```

golangci-lint (1.64.8)

var `testDeviceTokenGlobal` is unused

(unused)